v2/client_test.go

```diff
@@ -125,6 +125,7 @@ func TestACL_Unmarshal(t *testing.T) {
 			Name:       "It should handle HuJSON ACLs",
 			ACLContent: huJSONACL,
 			UnmarshalFunc: func(b []byte, v interface{}) error {
+				b = append([]byte{}, b...)
```

Comment:
Should we fix this in v1 too?

Reply:
@mpminardi already did - https://github.com/tailscale/tailscale-client-go/blob/main/tailscale/client_test.go#L128